webapp/graphite/render/functions.py

```diff
@@ -5102,7 +5102,10 @@ def applyByNode(requestContext, seriesList, nodeNum, templateFunction, newName=N
   """
   prefixes = set()
   for series in seriesList:
-    prefix = '.'.join(series.name.split('.')[:nodeNum + 1])
+    nodes = series.name.split('.')
+    if nodeNum >= len(nodes):
```

Comment:
```suggestion
    if nodeNum > len(nodes):
```

Reply:
My mistake. List in Python have len() method. This fix (len function) for Golang.